react/src/components/Table/TableFilter/ColumnFilter.tsx

```diff
@@ -27,6 +28,17 @@ export const ColumnFilter: React.FC<ColumnFilterProps> = ({
     setFilter,
     type,
 }) => {
+    const [debouncedInput, input, setInput] = useDebounce<string | undefined>('', 500);
```

Comment:
This seems to be working as expected on the regular text inputs. However, the number range filters seem to be functioning as before -- input is passed straight to the filters w/o debounce.

Reply:
That's right. It's because I didn't pass in the debounce input to the number range component. I can add that.